.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -0,0 +1,92 @@
+  
+jobs:
+  process-sdk-suppressions-labels:
+    name: Process Sup
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    # outputs:
+      # process-sdk-suppressions-output: ${{ steps.run-suppressions-script.outputs.suppressions-result-output }}
+      # has-output: ${{ steps.run-suppressions-script.outputs.has-output }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          # Required since "HEAD^" is passed to Get-ChangedFiles
+          fetch-depth: 2
+
+      - name: Setup Node and run `npm ci`
+        uses: ./.github/actions/setup-node-npm-ci
+
+      - name: Get GitHub PullRequest Changed Files
+        shell: pwsh
+        id: get-changedFiles
+        run: |
+          . eng/scripts/ChangedFiles-Functions.ps1
+          $changedFiles = @(Get-ChangedFiles)
+          echo "PR Changed files: $changedFiles"
+          echo "::set-output name=changedFiles::$changedFiles"
+
+      - name: Get GitHub PullRequest Context
+        uses: actions/github-script@v7
+        id: fetch-pullRequest-context
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            if (!pr) {
+              throw new Error("This workflow must run in the context of a pull request.");
+            }
+            console.log("This action trigger by ", context.eventName);
+            const prContext = {
+              number: pr.number,
+              title: pr.title,
+              html_url: pr.html_url,
+              base: {
+                owner: pr.base.repo.owner.login,
+                repo: pr.base.repo.name,
+                ref: pr.base.ref
+              },
+              head: {
+                owner: pr.head.repo.owner.login,
+                repo: pr.head.repo.name,
+                ref: pr.head.ref
+              },
+              labels: pr.labels.map(label => label.name)
+            };
+            return JSON.stringify(prContext);
+          result-encoding: string
+
+      - name: Run get suppressions label script
+        id: run-suppressions-script
+        env:
+          GITHUB_PULL_REQUEST_CONTEXT: ${{ steps.fetch-pullRequest-context.outputs.result }}
+          OUTPUT_FILE: "output.json"
+          GITHUB_PULL_REQUEST_CHANGE_FILES: ${{ steps.get-changedFiles.outputs.changedFiles }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          node eng/tools/sdk-suppressions/cmd/sdk-suppressions-label.js
+          OUTPUT=$(cat $OUTPUT_FILE)
+          echo "Script output labels: $OUTPUT"
+          echo "::set-output name=suppressions-result-output::$OUTPUT"
+          
+          has_output=$(node -e "const fs = require('fs'); const data = JSON.parse(fs.readFileSync('$OUTPUT_FILE', 'utf8')); console.log(data.length === 0 ? 'false' : 'true');")
+          echo "Script has-output: $has_output"
+          echo "::set-output name=has-output::$has_output"
+          
+      - name: Upload artifact dependes on labels
```

Comment:
I discussed this with @raych1.  Assuming the list of labels is static and relatively short, I think the best option is to just use multiple calls to `add-label-artifact`, with the name of each label hardcoded in the YML:

```
- uses: ./.github/actions/add-label-artifact
  name: Upload artifact for label "foo"
  with:
    name: "foo"
    value: "${{ steps.mystep.outputs.foo == 'true' }}"

- uses: ./.github/actions/add-label-artifact
  name: Upload artifact for label "bar"
  with:
    name: "bar"
    value: "${{ steps.mystep.outputs.bar == 'true' }}"
```

The root cause is the `actions/upload-artifact` action provided by GitHub only supports a single name, and YML doesn't have any native looping constructs.